### DISCUSSION TITLE
extract value

### DISCUSSION THREAD

**Discussion by syreccop** [deepmodeling/deepmd-kit#4399] [Q&A]
Is it possible to obtain the attention weight in DPA-1 or DPA-2?

**Comment by njzjz** [comment]
You can convert a model file to a YAML file using `dp convert-backend`.

**Reply by syreccop** [reply]
Is there any specific version I should use?
I failed using v3.0.0rc0

`Traceback (most recent call last): File "/usr/local/bin/dp", line 8, in <module> sys.exit(main()) File "/usr/local/lib/python3.10/dist-packages/deepmd/main.py", line 927, in main deepmd_main(args) File "/usr/local/lib/python3.10/dist-packages/deepmd/entrypoints/main.py", line 86, in main convert_backend(**dict_args) File "/usr/local/lib/python3.10/dist-packages/deepmd/entrypoints/convert_backend.py", line 26, in convert_backend data = inp_hook(INPUT) File "/usr/local/lib/python3.10/dist-packages/deepmd/tf/utils/serialization.py", line 47, in serialize_from_file model_dict = model.serialize() File "/usr/local/lib/python3.10/dist-packages/deepmd/tf/model/model.py", line 901, in serialize "descriptor": self.descrpt.serialize(suffix=suffix), File "/usr/local/lib/python3.10/dist-packages/deepmd/tf/descriptor/se_atten_v2.py", line 189, in serialize data = super().serialize(suffix) File "/usr/local/lib/python3.10/dist-packages/deepmd/tf/descriptor/se_atten.py", line 1928, in serialize raise RuntimeError( RuntimeError: The implementation for smooth_type_embedding is inconsistent with other backends`

**Reply by njzjz** [reply]
You can comment out this line.

**Reply by syreccop** [reply]
Sorry which one should I comment out? I tried but still failed
But I can convert the backend of a model to `.yaml` if I use descriptor such as `se_e2_a`
